lib/rails_utils.rb

```diff
@@ -3,25 +3,15 @@
 
 module RailsUtils
   module ActionViewExtensions
-    def page_controller_class
-      case RailsUtils.configuration.selector_format
-      when :underscored
-        page_controller_class_underscored
+    def page_controller_class(type: RailsUtils.configuration.selector_format)
```

Comment:
Actually I prefer the previous implementation of having two methods call `page_controller_class_hyphenated` and `page_controller_class_underscored`. It's clearer that way.